src/pages/auth/sign-in.tsx

```diff
@@ -48,7 +48,7 @@ export const getServerSideProps: GetServerSideProps<PageProps> = async () => {
         github:
           env.AUTH_GITHUB_CLIENT_ID !== undefined &&
           env.AUTH_GITHUB_CLIENT_SECRET !== undefined,
-        credentials: true,
+        credentials: env.NEXT_PUBLIC_INTERNAL_CREDENTIALS_DISABLED !== "true",
```

Comment:
Rename to AUTH_SOMETHING as it's only used in server and props

Reply:
[716807b](https://github.com/langfuse/langfuse/pull/510/commits/716807b6c110dd1f9d55ab9148cfaeb6e70473be)